intermediate_html_css/intermediate_css_concepts/browser_compatibility.md

```diff
@@ -20,7 +20,7 @@ The release of Nexus was just the beginning though, as in the next decade people
 
 In 1995 the world got introduced to the first version of Internet Explorer, which became the dominant player in the market. At some point, Internet Explorer was used by more than 90% of all users. To counter this dominance, Netscape launched what would become Mozilla Foundation which develops and maintains Firefox. Soon after that, in 2003, Apple launched Safari, and in 2008, Google launched Chrome.
 
-You're most likely familiar with most, if not all these names.<span id="most-used-browser"> There is a lot of competition among browsers still to this day, even though Chrome (and [Chromium](https://en.wikipedia.org/wiki/Chromium_(web_browser))) is the dominant player in the market</span>.
+You're most likely familiar with most, if not all these names.<span id="most-used-browser"> There is a lot of competition among browsers still to this day (as seen in [this infographic video](https://www.youtube.com/watch?v=W4wWdmfOibY)), even though Chrome (and [Chromium](https://en.wikipedia.org/wiki/Chromium_(web_browser))) is the dominant player in the market</span>.
```

Comment:
For accessibility, we need to make sure [links have descriptive labels without "this" or "here"](https://github.com/TheOdinProject/curriculum/blob/main/LAYOUT_STYLE_GUIDE.md#use-informative-titles). Otherwise, users with screen readers navigating by links would simply get "this infographic video - link" read out to them, which does not provide sufficient context for what the linked resource is about or what site it leads to.

Below is a minimally invasive suggestion that should provide enough description for the link.

```suggestion
You're most likely familiar with most, if not all these names.<span id="most-used-browser"> There is a lot of [competition among browsers](https://www.youtube.com/watch?v=W4wWdmfOibY) still to this day, even though Chrome (and [Chromium](https://en.wikipedia.org/wiki/Chromium_(web_browser))) is the dominant player in the market</span>.
```